train.py

```diff
@@ -182,19 +191,12 @@ def train(args, loader, generator, discriminator, g_optim, d_optim, g_ema, devic
     else:
         g_module = generator
         d_module = discriminator
+    normalisation_data = {'std': torch.tensor(std_joints.transpose(0, 2, 1, 3)).cuda(),
+                          'mean': torch.tensor(mean_joints.transpose(0, 2, 1, 3)).cuda()}
 
-    accum = 0.5 ** (32 / (10 * 1000))
-
-    report_every = args.report_every
     time_measure = []
     start_time_measure = time.time()
-    for idx in pbar:
-        i = idx + args.start_iter
-
-        if i > args.iter:
-            print("Done!")
-            break
-
+    for i in pbar:
```

Comment:
why did you delete the lines that deal with start_iter? they enable utilizing an existing model

Reply:
I didn't delete them, they appear in the definition of pbar itself a few line earlier.
This feature still exists - before that fix the tqdm prints were not correct.